op-chain-ops/genesis/layer_two_test.go

```diff
@@ -94,5 +94,5 @@ func TestBuildL2MainnetNoGovernanceGenesis(t *testing.T) {
 	config.EnableGovernance = false
 	config.FundDevAccounts = false
 	gen := testBuildL2Genesis(t, config)
-	require.Equal(t, 2324, len(gen.Alloc))
+	require.Equal(t, 2326, len(gen.Alloc))
```

Comment:
Why did this increase? We only moved the predeploys, but didn't add more.

Reply:
Internally current OP genesis will generate predeploys for 2048 0x4200* addresses (it will not automatically generate anything for these addresses by default, but it will add them to Alloc variable). Since we moved two smart contracts from this range our mapping size was increased.